app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt

```diff
@@ -139,7 +139,7 @@ class HelpFragmentTest {
   }
 
   @Test
-  fun openHelpActivity_configurationChanged_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() { // ktlint-disable max-line-length
+  fun openHelpActivity_configChanged_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() {
```

Comment:
``showsFaq`` (see: https://developer.android.com/kotlin/style-guide#camel_case for camel casing rules).

Reply:
Done!